wasmtime-jni/src/io_github_kawamuray_wasmtime_Module/imp.rs

```diff
@@ -16,6 +20,70 @@ impl<'a> JniModule<'a> for JniModuleImpl {
         Ok(())
     }
 
+    #[allow(unreachable_code)]
+    fn imports<'b>(
+        env: &'b JNIEnv,
+        this: JObject,
+    ) -> std::result::Result<jobjectArray, Self::Error> {
+        const STRING_CLASS: &str = "java/lang/String";
+        const IMPORT_TYPE: &str = "io/github/kawamuray/wasmtime/ImportType";
+
+        let module = interop::get_inner::<Module>(env, this)?;
+        let it = module.imports();
+        let mut imports = Vec::with_capacity(it.len());
+        for (_, obj) in it.enumerate() {
+            let module = obj.module();
+            let [ty, ty_obj] = match obj.ty() {
```

Comment:
I think this works, but much idiomatic way is to use tuple instead: `let (ty, ty_obj) = match ... { (a, b) }`